vignettes/my-vignette.Rmd

```diff
@@ -0,0 +1,216 @@
+
+## Update Settings
+
+After loading your data, navigate to the Settings tab to customize the behavior of the charts. This tab includes panels for different types of chart settings.  For example, the "Data Mapping" panel (shown below for the "Example Data" ADaM data set) can be used to specify the column that contains the unique subject ID, and on the more general "Appearance Settings" panel, there is an option to specify a warning message to be displayed when the chart loads. You can hover the mouse over any setting label to get more details. 
+
+<img src="https://user-images.githubusercontent.com/3680095/51495648-f1fe5e00-1d71-11e9-97e6-939ef6d08a9b.png"  width="80%">
+
+When possible, items on the settings tab are pre-populated based on the data standard of the selected data set. See the Case Studies below for more details regarding working with non-standard data and adding customizations to the charts. 
+
+## View Chart
+
+Once the settings configuration is complete and a green check is shown in the navigation bar, navigate to the chart tab to view the chart. The chart tab updates automatically when settings are changed or new data is loaded. 
+More details about chart functionality will be documented in separate vignettes.  
+
+<img src="https://user-images.githubusercontent.com/3680095/51504626-c8a3f900-1d96-11e9-91d5-3641378844f1.png"  width="80%">
+
+## Export Results
+
+Finally, click the "Export Chart" button in the upper right corner to create a standalone copy of the chart using the current configuration. The export functionality combines the data, code, and settings for the chart in to a single file. In addition to the chart itself, the export includes a summary of the tool, and code to recreate the customized chart in R. 
+
+<img src="https://user-images.githubusercontent.com/3680095/51543124-b61ad580-1e11-11e9-8fe2-fe1a67a83583.png"  width="80%">
+
+# Case Study #1 - Mapping Non-Standard data
+
+## Overview
+
+When a new data file is loaded, the app will detect whether the dataset is formatted according to ADaM or SDTM data standards.  If the uploaded dataset matches one of these standards, the settings tab will be pre-populated accordingly, and little or no custom user customization will be needed to generate a basic chart. However, no data standard is strictly required; the app also works with data in other formats. The only firm data requirements for the eDish chart are:
+
+- The data must have one record per participant per timepoint per lab test. That is, the data should be long, not wide.
+- The data must have columns for: 
+  - Unique Subject Identifier (ID Column)
+  - Name of Measure or Test (Measure Column)
+  - Numeric finding or result (Value Column)
+  - Numeric upper limit of normal for the test
+  - Numeric lower limit of normal for the test
+  - Numeric study day
+- The following values must be present in the column corresponding to Measure or Test:
+  - Aminotransferase, alanine (ALT)
```

Comment:
Bullet points for ALT, AST, ALP, and TB need an extra indentation 
![image](https://user-images.githubusercontent.com/14199771/51913864-83d71e00-23a5-11e9-9f8d-a41b314411f1.png)

Reply:
good catch. will fix.